woocommerce/woocommerce
# [testing workflows] Include more paths in changes lists

### Changes proposed in this Pull Request:

Closes https://github.com/woocommerce/woocommerce/issues/50374

Update the changes lists for some test jobs so they get triggered in CI.

### How to test the changes in this Pull Request:

1. Create a branch from this PRs branch (`ci/include-woocommercephp-in-changes`)
2. Change a relevant file
3. Run `pnpm utils ci-jobs -r ci/include-woocommercephp-in-changes`
4. Check the output
